mysql/toddl.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mysql
+
+import (
+	"fmt"
+	"strconv"
+	"unicode"
+
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/schema"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+)
+
```

Comment:
I think this file is identical to ../postgres/toddl.go, except for toSpannerType.
We should refactor at some point to avoid the duplication.

Let's add a TODO.

// TODO: refactor this file to avoid the duplication with postgres/toddl.go.
// The core difference between the two files is toSpannerType, which maps
// type ids (which differ between MySQL and PostgreSQL) to Spanner types.

Reply:
Done.